cmd/injective-price-oracle/oracle.go

```diff
@@ -201,12 +203,52 @@ func oracleCmd(cmd *cli.Cmd) {
 			log.Infof("found %d dynamic feed configs", len(dynamicFeedConfigs))
 		}
 
+		storkFeedConfigs := make([]*oracle.StorkFeedConfig, 0, 10)
+		if len(*storkFeedsDir) > 0 {
+			err := filepath.WalkDir(*storkFeedsDir, func(path string, d fs.DirEntry, err error) error {
+				if err != nil {
+					return err
+				} else if d.IsDir() {
+					return nil
+				} else if filepath.Ext(path) != ".toml" {
+					return nil
+				}
+
+				cfgBody, err := ioutil.ReadFile(path)
+				if err != nil {
+					err = errors.Wrapf(err, "failed to read stork feed config")
+					return err
+				}
+
+				feedCfg, err := oracle.ParseStorkFeedConfig(cfgBody)
```

Comment:
we should support price updates for multiple tickers

Reply:
Updated flow, can you plz recheck?